### DISCUSSION TITLE
Pull Shark Badge

### DISCUSSION THREAD

**Discussion by M9yk** [community/community#18758] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by jeezou** [comment, marked as answer]
This badge is given if two of your pull requests were accepted

https://github.com/Schweinepriester/github-profile-achievements

**Comment by mah-shamim** [comment]
This badge is given if two of your pull requests were accepted

https://github.com/Schweinepriester/github-profile-achievements

**Comment by tamer1an** [comment]
We like it. :rocket:

**Comment by the-j0k3r** [comment]
Well, this isnt very accurate at all, shows only 3 Pr's merged for me, when in fact there has been way more than 3.
